### ISSUE TITLE
asr_based_vad_threshold isn't what it's supposed to be

### ISSUE THREAD

**Issue by roman-vygon** [NVIDIA/NeMo#4104]
**Describe the bug**

In [asr_with_diarization config](https://github.com/NVIDIA/NeMo/blob/2aa68822bbfb8e7e8c221817b66de28928d6443b/examples/speaker_tasks/diarization/conf/offline_diarization_with_asr.yaml) the threshold is described as follows:
```yml
parameters:
      asr_based_vad: False # if True, speech segmentation for diarization is based on word-timestamps from ASR inference.
      asr_based_vad_threshold: 50 # threshold (multiple of 10ms) for ignoring the gap between two words when generating VAD timestamps using ASR based VAD.
```

However in [ASR_DIAR_OFFLINE.get_speech_labels_from_decoded_prediction](https://github.com/NVIDIA/NeMo/blob/2aa68822bbfb8e7e8c221817b66de28928d6443b/nemo/collections/asr/parts/utils/diarization_utils.py#L185) it is used as a value in seconds:
```python
  if word_ts[count][0] - word_ts[count - 1][1] <= self.nonspeech_threshold:
```

So, the default config defines minimum distance between words as 50 seconds, which usually results in the vad manifest being just one line with start and end time of the audio file.

**Steps/Code to reproduce bug**

```python
from nemo.collections.asr.parts.utils.diarization_utils import ASR_DIAR_OFFLINE
import json

with open("test_manifest.json",'w') as f:
    f.write(json.dumps({'audio_filepath':"test.wav"}))
cfg_diarizer = {
    "manifest_filepath":"test_manifest.json",
    "asr":{"parameters":{"asr_based_vad_threshold":50,"fix_word_ts_with_VAD":False},"ctc_decoder_parameters":{},
          "realigning_lm_parameters":{}},    
    
    "out_dir":"."
}
asr_diar_offline = ASR_DIAR_OFFLINE(**cfg_diarizer)

fake_ts = {'test.wav': [[0.0, 20.], [40., 45.]]}


asr_diar_offline.save_VAD_labels_list(fake_ts)
!cat {asr_diar_offline.VAD_RTTM_MAP['test.wav']['rttm_filepath']}
```

**Expected behavior**
Two time ranges 0-20 and 40-45 are split in the vad rttm

**Comment by titu1994** [comment]
@tango4j can you take a look ?

**Comment by tango4j** [comment]
Hi. Thank you for pointing out this error. Yes, it should be in second, not in multiple of 10ms. I guess the Yaml file has not been updated by mistake. I have opened a [PR](https://github.com/NVIDIA/NeMo/pull/4141) for fixing the wrong description in yaml file.

**Comment by roman-vygon** [comment]
fixed in #4141
